src/pages/organize/[orgId]/campaigns/[campId]/calendar.tsx

```diff
@@ -42,27 +48,53 @@ export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
     }
 }, scaffoldOptions);
 
-type CampaignCalendarPageProps = {
+type OrganizeCalendarPageProps = {
     campId: string;
     orgId: string;
 };
 
-const CampaignCalendarPage: PageWithLayout<CampaignCalendarPageProps> = ({ orgId, campId }) => {
+const CampaignCalendarPage : PageWithLayout<OrganizeCalendarPageProps> = ({ orgId, campId }) => {
     const eventsQuery = useQuery(['campaignEvents', orgId, campId], getCampaignEvents(orgId, campId));
     const events = eventsQuery.data || [];
+    const intl = useIntl();
+
+    const items = [
+        {
+            name: intl.formatMessage(
+                { id: 'misc.calendar.month' }),
+        },
+        {
+            name: intl.formatMessage(
+                { id: 'misc.calendar.week' }),
+        },
+    ];
```

Comment:
I would add some sort of persistent (compile-time), machine-readable key to these. I'm not sure what `<Picker>` expects, but maybe `key` or `id`? Or maybe you can use any name for the field here and then pass it to the right property of `<Item>` (probably the `key` property?).

In fact I usually use dictionaries for this, instead of arrays, like so:

```typescript
const items = {
  month: intl.formatMessage({ id: 'misc.calendar.month' }),
  week: intl.formatMessage({ id: 'misc.calendar.week' }),
};
```